dev/NavigationView/NavigationView.xaml

```diff
@@ -346,13 +346,16 @@
                                         VerticalContentAlignment="Stretch"
                                         HorizontalContentAlignment="Stretch"
                                         Grid.Column="7" />
-
-                                    <local:NavigationViewItem
+                                    <Grid
+                                        Grid.Column="8"
+                                        >
```

Comment:
Just formatting, '>' should be on the previous line

Reply:
Thanks for pointing that out! Fixed that with latest commit.

Reply:
Awesome! :)